platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/PlatformData.java

```diff
@@ -502,7 +502,7 @@ public String toString() {
                 .append("hashEventsCons", hashEventsCons)
                 .append("events", events)
                 .append("consensusTimestamp", consensusTimestamp)
-                .append("minGenInfo", minGenInfo)
+                .append("minGenInfo", minimumJudgeInfo)
```

Comment:
fieldName should be updated to match new member variable name

Reply:
fixed